dev/AppLifecycle/ActivationRegistrationManager.h

```diff
@@ -9,7 +9,8 @@ namespace winrt::Microsoft::Windows::AppLifecycle::implementation
     // Registration constant values.
     static PCWSTR c_argumentPrefix{ L"----" };
```

Comment:
use `const` or `constexpr` instead of `static`. statis results in unique copies of the variables in every translation unit that includes it.